src/java-opentimelineio/src/main/java/io/opentimeline/opentimelineio/Deserialization.java

```diff
@@ -0,0 +1,11 @@
+package io.opentimeline.opentimelineio;
+
+public class Deserialization {
+
+    public static native boolean deserializeJSONFromString(
```

Comment:
consider these methods not being static

Reply:
@margant is there a reason to make these `not static`?

In c++ these are not inside any class. Just in the OTIO namespace.

Reply:
Mainly in Java you'd use static methods for Library functions (Apache String Utils is one example) or for getInstance() of the Singleton, where the nature requires the static. One of the major downside of using static is not having an option to use dependency injection for upstream classes that would take a "mock" of the Serialization/Deserialization rather then the actual implementation. Most enterprise frameworks utilize DI heavily, so static methods are not too common and if used are very purposeful.

Reply:
Oh. Although I've heard of it but I am not familiar with DI. This is new for me. Thanks @margant! I'll remove static.